item-spec/item-spec.md

```diff
@@ -252,6 +218,48 @@ in the file).
 For examples of fields that this construct is recommended for, see the [section of STAC Best Practices](../best-practices.md#common-use-cases-of-additional-fields-for-assets)
 that talks about common use cases of additional fields for assets.
 
+## Media Types
+
+One of the best ways to help inform web clients about the content in a link is to use a common [Media 
+Type](https://en.wikipedia.org/wiki/Media_type) in the `type` field. In STAC the `type` field is used in both the 
+`[Link](#link-object)` and `[Asset](#asset-object)` Objects. It is quite useful for STAC browsers to better determine
+what to render and display to users searching and browsing the catalog.  Media types are often referred to by the 
+now-deprecated term "MIME types". 
+
+Any media type can be used in STAC, and [registered](https://www.iana.org/assignments/media-types/media-types.xhtml) 
+Media Types are preferred. In cases where custom vendor-specific media types are necessary, they should
+use the `vnd.` prefix, see [RFC 6838 section-3.2](https://tools.ietf.org/html/rfc6838#section-3.2). STAC Items that 
+have sidecar metadata files associated with a data asset (e.g, `.tfw`, Landsat 8 MTL files)
+should use media types appropriate for the the metadata file.  For example, if it is a plain text file, then `text/plain`
+would be appropriate; if it is an XML, then `text/xml` is appropriate. For more information on media types as well as a 
+list of [common media types](../best-practices.md#common-media-types-in-stac) used in STAC see the [best practice on 
+working with media types](../best-practices.md#working-with-media-types).
+
+### STAC Media Types
+
+The following table lists the Media Types for STAC, as well as the way [GeoTIFF](https://en.wikipedia.org/wiki/GeoTIFF)'s 
+(and [COG](https://www.cogeo.org)'s) are referenced in STAC.
+
+| Media Type                                              | Description                                                  |
```

Comment:
It's a bit strange that the Item Spec has the media types for all "specs", but Collections and Catalogs, where you'd need them first in the hierarchy, have no such table. So I don't think this is the right place to put such a table. This is better suited in catalog actually as there you link to items and catalogs (and collection inherits from there). In Items you don't link to items and catalogs at all.

Reply:
Good points. I'm not even sure why I did it in items. I agree Catalog seems to be the most appropriate spot for it. Will move it.

Reply:
> Finally, if there's really just GeoTiff/COG left, also move it to the best practice. It seems strange to just have them listed here in its own chapter.

Yeah, I think I was compelled originally by the idea that 'we're doing our own thing', and it's worth calling out. But I agree it feels a bit weird like this. I'll put it in the best practice. Could be nice to 'set' the mime type somewhere for COG, like in the 'cog spec' or on cogeo.org.